Pinia nuxt version 0.6.0+ causes error when used with pnpm monorepo

### Reproduction

https://stackblitz.com/~/github.com/kalvenschraut/pinia-nuxt-pnpm-monorepo-issue

### Steps to reproduce the bug

To see issue do
```shell
pnpm install
pnpm --filter website1 run dev
```

can then downgrade @pinia/nuxt to 0.5.5 and run same commands to see it working. Can also upgrade to 0.6.1 to verify still not working
```shell
pnpm update @pinia/nuxt@0.5.5 -r
pnpm --filter website1 run dev
```


### Expected behavior

Page to load and show the users name from the user store

### Actual behavior

![image](https://github.com/user-attachments/assets/6c7edb2d-61c1-4d60-bd86-939bd24aabf5)


### Additional information

In prod builds on my personal site where I first initial saw this was seeing vueDemi.effectScope is not a function error. Downgrading back to 0.5.5 for now until direction can be given

Hello, today I've updated from v0.5.5 to v0.7.0 and couldn't run the project.

This is the error I'm receiving "[nuxt] [request error] [unhandled] [500] Cannot find module '/.output/server/node_modules/pinia/dist/pinia.mjs' imported from /.output/server/chunks/build/server.mjs"

and when I check the file in node module there is only pinia.prod.cjs file.

nuxt: 3.13.2

also I use pnpm

Thank you

--- a/packages/nuxt/package.json
+++ b/packages/nuxt/package.json
@@ -46,7 +46,9 @@
     "changelog": "conventional-changelog -p angular -i CHANGELOG.md -s --commit-path . -l @pinia/nuxt -r 1"
   },
   "dependencies": {
-    "@nuxt/kit": "^3.14.1592",
+    "@nuxt/kit": "^3.14.1592"
+  },
+  "peerDependencies": {
     "pinia": "workspace:^"
   },
   "devDependencies": {
@@ -54,6 +56,7 @@
     "@nuxt/schema": "^3.14.1592",
     "@nuxt/test-utils": "^3.14.4",
     "nuxt": "^3.14.1592",
+    "pinia": "workspace:^",
     "typescript": "^5.6.3",
     "vue-tsc": "^2.1.10"
   },

